cloudfront/v0.0.1/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "cloudfront_arn" {
+  description = "arn of cloudfront"
+  value       = aws_cloudfront_distribution.cf_dist.arn
+}
```

Comment:
Please add an output for the DNS name of the cloudfront distro that's created so we can use it for Route53 records